recipe/meta.yaml

```
@@ -34,13 +32,68 @@ test:
     - udunits2 -h
     - udunits2 -H meter -W miles
 
+outputs:
+  - name: libudunits2
+    script: install.sh  # [unix]
+    script: install.bat  # [win]
+    files:
+      - lib/libudunits2.so.*  # [linux]
+      - lib/libudunits2.*.dylib  # [osx]
```

```suggestion
      - lib/libudunits2.*dylib  # [osx]
```
Similar issue here, I think.